pkgs/data/fonts/camingo-code/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "camingo-code-${version}";
+  version = "1.0";
+
+  src = fetchurl {
+    url = https://github.com/chrissimpkins/codeface/releases/download/font-collection/codeface-fonts.zip;
+    sha256 = "1gbpfa5mqyhi5yrb6dl708pggiwp002b532fn3axiagb0cxxf02s";
+  };
+
+  buildInputs = [ unzip ];
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/truetype
+    mkdir -p $out/share/doc/${name}
+    cp -v camingo-code/*.ttf $out/share/fonts/truetype/
+    cp -v camingo-code/*.txt $out/share/doc/${name}/
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
There doesn't seem to be an entry for this license (http://creativecommons.org/licenses/by-nd/3.0) in licenses.nix. Should I add it there?